### ISSUE TITLE
Crash on Attempt to Mod Config 

### ISSUE THREAD

**Issue by Stickly10** [neoforged/NeoForge#1357]
**Minecraft Version:** 1.21

**NeoForge Version:** 21.0.113-beta

**Logs:** (https://pastebin.com/heifctmA)

**Steps to Reproduce:**
 1. Launch Minecraft with NeoForge 21.0.113-beta.
 2. Add any mod (log shows xaeros, but it still happens with any other mod).
 3. Attempt to open a mod's configuration screen.
 4. Observe the crash.

**Description of issue:**
I'm encountering a persistent crash related to `java.lang.AbstractMethodError` when trying to use NeoForge 21.0.113-beta with Minecraft 1.21. The crash occurs when interacting with the Mod List Screen or trying to access the configuration screen of mods.

Error Message:
```
The game crashed whilst mouseclicked event handler
Error: java.lang.AbstractMethodError: Receiver class xaero.common.PlatformContextLoaderClientOnlyNeoForge$$Lambda/0x000000080111dc08 does not define or inherit an implementation of the resolved method 'abstract net.minecraft.client.gui.screens.Screen createScreen(net.neoforged.fml.ModContainer, net.minecraft.client.gui.screens.Screen)' of interface net.neoforged.neoforge.client.gui.IConfigScreenFactory.
```

**System Details:**
- Minecraft Version: 1.21
- NeoForge Version: 21.0.113-beta
- OS: Windows 10 (amd64) version 10.0
- Java Version: 21.0.3, Microsoft
- Graphics Card: NVIDIA GeForce RTX 4060 Ti
- JVM Flags: -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx4096m -Xms256m

**Comment by Matyrobbrt** [comment]
The mods that crash with that error need to update to a change introduced with the built-in config UI.

**Comment by Rheckar** [comment]
I have the same problem ever since I switched from forge to neoforge and any time I try to use the config button the game crashes no matter how many mods there are 1 or 70. I also can't run the game with the configured mod which could be the problem but the game gives an uncaught parallel processing error. Not sure what we can do about the update to the built-in config ui problem, how do we learn which mods are the problem? I'm positive this happens on every mod I have.

**Comment by HenryLoenwind** [comment]
To explain this in layman's terms:

Imagine you're building a house. You ask the architect for the plans, and he gives them to you but tells you they are not yet final and will still change a couple of times. You then order a custom-built kitchen that fits the plans perfectly to a tenth of an inch. Later, when the house is up and done, you try to install the kitchen, but it not only won't fit, the kitchen isn't even at the same corner of the house.

Until a couple of hours ago, NeoForge was that unfinished plan. So anything built to fit those unfinished plans has a good chance to not ft the final ones.
